lib/taiko.js

```diff
@@ -707,16 +708,25 @@ module.exports.attach = async (filepath, to) => {
  * @param {number} [options.timeout=5000] - Timeout value in milliseconds for navigation after click.
  * @returns {Promise<Object>} - Object with the description of the action performed.
  */
-module.exports.press = async (key, options = {}) => {
+module.exports.press = async (keys, options = {}) => {
     validate();
     options = setNavigationOptions(options);
+    return await _press(new Array().concat(keys), options);
+};
+
+async function _press(keys, options) {
     await doActionAwaitingNavigation(options, async () => {
-        await inputHandler.down(key, options);
-        if (options && options.delay) await new Promise(f => setTimeout(f, options.delay).unref());
-        await inputHandler.up(key);
+        for (let i = 0; i < keys.length; i++) {
+            await inputHandler.down(keys[i], options);
+            if (options && options.delay) await new Promise(f => setTimeout(f, options.delay).unref());
```

Comment:
The delay need not be in between each key down. It should be after all key down happens and before key up starts.